### ISSUE TITLE
Cannot subscribe to events in lambda view

### ISSUE THREAD

**Issue by lszymik** [kyma-project/kyma#3883]
**Description**

There is an issue which occurs on Kyma 1.0 (Minikube) and probably also on Kyma 0.9 (also Minikube)

**Expected result**

The lambda view should allow subscription to the events.

**Actual result**

```https://apiserver.kyma.local/apis/gateway.kyma-project.io/v1alpha2/namespaces/stage/apis/test```

This link return 404

<img width="2154" alt="Screenshot 2019-04-26 at 12 07 36" src="https://user-images.githubusercontent.com/8614653/56800733-0bed8800-681c-11e9-98a6-12ed61b2619d.png">

**Steps to reproduce**

- Register Event Catalog
- Create a lambda
- Subscribe event

**Comment by sayanh** [comment]
It found that, lambda-ui when queries core-console-backend-service
```
{"query":"query EventActivations($namespace: String!) {\n      eventActivations(namespace: $namespace) {\n        name\n        displayName\n        sourceId\n        events {\n          eventType\n          version\n          description\n        }\n      }\n    }","variables":{"namespace":"stage"}}
```
the response is as follows:
```
{"errors":[{"message":"internal error","path":["eventActivations",0,"events"]}],"data":{"eventActivations":[{"name":"3681fc47-8407-4800-9f48-fdad0dc0eaf5","displayName":"ec-all-eventstest","sourceId":"commerce","events":null}]}}
```

As there are no events in the response, the list of events in Add event trigger is empty. On debugging further, I found core-console-backend-service pod is unable to talk to application connector gateway:
```
Application commerce health check failed (commerce-application-gateway-external-api.kyma-integration.svc.cluster.local:8081), error: Get http://commerce-application-gateway-external-api.kyma-integration.svc.cluster.local:8081/v1/health: read tcp 192.168.64.2:37586->10.111.245.176:8081: read: connection reset by peer
```

This is solely due to [this](https://github.com/kyma-project/kyma/issues/3841) issue.
CCing @derberg

**Comment by derberg** [comment]
Related issues are fixed, fix is in 1.1 already
